public/components/NotificationsCallOut/NotificationsCallOut.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { EuiCallOut, EuiLink, EuiSpacer } from '@elastic/eui';
+
+export const NotificationsCallOut = () => {
+  return (
+    <div>
+      <EuiCallOut title="Notifications plugin is not installed" color="danger" iconType="alert">
+        <p>
+          Install the notifications plugin in order to create and select channels to send out
+          notifications.{' '}
```

Comment:
Should we use `&nbsp;` instead?

Reply:
Good point, just used the same component from alerting.

Fixed